Makefile

```diff
@@ -6,6 +6,7 @@ quality:
 	black --check --line-length 119 --target-version py35 examples templates tests src utils
 	isort --check-only --recursive examples templates tests src utils
 	flake8 examples templates tests src utils
+	python utils/check_repo.py
```

Comment:
is it fast?

Reply:
Takes less than a second, so faster than all the other three.